examples/gltf/app.js

```diff
@@ -268,7 +270,7 @@ export class DemoApp {
     };
   }
 
-  onInitialize({gl, canvas}) {
+  onInitialize({gl, canvas, _loop: animationLoop}) {
```

Comment:
Nit; I think you want to put long helper methods like `initializeEventHandling` last in the class to make it easier for readers to "scan".

Reply:
Nit: Any chance you can use `async/await` instead of `then` below?